packages/plugin-ext/src/plugin/notification.ts

```diff
@@ -48,8 +48,12 @@ export class NotificationExtImpl implements NotificationExt {
         return progress;
     }
 
-    protected mapLocation(pluginLocation: ProgressLocation): string | undefined {
-        switch (pluginLocation) {
+    protected mapLocation(location: ProgressLocation | { viewId: string }): string | undefined {
+        if (typeof location === 'object' && location.viewId) {
```

Comment:
VS Code [does not do a check](https://github.com/microsoft/vscode/blob/7d1affcf90de1559b6d6f8da01eac75d3b735750/src/vs/workbench/api/common/extHostTypeConverters.ts#L1145-L1147) for `location.viewId`.
What should happen when I pass in the following argument: `{ viewId: '' }`? It will evaluate to `falsy` and we hit the `switch`. Can you please remove the extra `location.viewId` check and align the code with Code 😊 ?

Reply:
thanks, I removed additional check for `location.viewId`